docs/analyze.rst

```diff
@@ -0,0 +1,385 @@
+
+      scenario = Scenario(3287)
+      capacity = get_capacity_factor_time_series(
+        scenario, "Texas", {"solar", "wind"}, area_type= "interconnect"
+      )
+
+- get the total generation for each generator type and load zone combination
+
+  .. code-block:: python
+
+      from powersimdata import Scenario
+      from postreise.analyze.generation.summarize import sum_generation_by_type_zone
+
+      scenario = Scenario(3101)
+      generation = sum_generation_by_type_zone(scenario)
+
+- get the total generation for each generator type and state combination, adding
+  totals for the interconnects and for all states
+
+  .. code-block:: python
+
+      from powersimdata import Scenario
+      from postreise.analyze.generation.summarize import sum_generation_by_state
+
+      scenario = Scenario(3101)
+      generation = sum_generation_by_state(scenario)
+
+
+- get the total historical generation for each generator type and state combination,
+  adding totals for interconnects and for all states
+
+  .. code-block:: python
+
+      import inspect
+      import os
+      import pandas as pd
+
+      import postreise
+      from postreise.analyze.generation.summarize import summarize_hist_gen
+
+      data = os.path.join(os.path.dirname(inspect.getfile(postreise)), "data")
+        hist_gen = pd.read_csv(
+        os.path.join(data, "2016_Historical_USA_TAMU_Generation_GWh.csv"), index_col=0
+      ).T
+      historical_generation = summarize_hist_gen(hist_gen, hist_gen.columns.to_list())
+
+- get hourly total generation for generator type(s) in an area
+
+  .. code-block:: python
+
+      from powersimdata import Scenario
+      from postreise.analyze.generation.summarize import (
+          get_generation_time_series_by_resources,
+      )
+
+      scenario = Scenario(3287)
+      generation = get_generation_time_series_by_resources(
+        scenario, "Western", {"solar", "wind"}
+      )
+
+- get hourly total storage power generation in an area
+
+  .. code-block:: python
+
+      from powersimdata import Scenario
+      from postreise.analyze.generation.summarize import get_storage_time_series
+
+      scenario = Scenario(1171)
+      generation = get_storage_time_series(scenario, "Western")
+
+
+Emission
+++++++++
+- calculate hourly emissions for each generator
+
+  .. code-block:: python
+
+      from powersimdata import Scenario
+      from postreise.analyze.generation.emissions import generate_emissions_stats
+
+      scenario = Scenario(403)
+      emission = generate_emissions_stats(scenario, pollutant="carbon", method="simple")
+
+  Different methods can be used to infer the carbon emission:
+
+    - 'simple' uses a fixed ratio of :math:`\rm{CO}_2` to MWh
+    - 'always-on' uses generator heat-rate curves including non-zero intercepts
+    - 'decommit' uses generator heat-rate curves but de-commits generators if off
+
+  Only the 'simple' method can be used to infer emission of :math:`\rm{NO}_x` or
+  :math:`\rm{SO}_2`.
+
```

Comment:
I think it would help to add what the pollutant call names are for NOx and SO2 when using the 'simple' method.
"carbon" = CO2
??? = NOx
??? = SO2

Reply:
Good idea
NOx = Nitrogen oxide
SO2=Sulfur dioxide

Reply:
I think @danlivengood means the parameters, in which case they're `"nox"` and `"so2"`

Reply:
Indeed, I meant what @danielolsen wrote

Reply:
Makes sense